test/controllers/shifts_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class ShiftsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @shift = shifts(:one)
+  end
+
+  test "should get index" do
+    get shifts_url
+    assert_response :success
+  end
+
+  test "should get new" do
+    get new_shift_url
+    assert_response :success
+  end
+
+  test "should create shift" do
```

Comment:
I think you should add some tests for the unhappy paths (eg if a date in the past is passed in during a `shift` creation)

Reply:
I will need to research the testing part. The task for the next few days 😅

Reply:
@GabrielleEaston Sounds good! Let me know if you need any help!